apps/ensrainbow/src/lib/database.ts

```diff
@@ -178,26 +193,30 @@ export class ENSRainbowDB {
   }
 
   /**
-   * Check if an ingestion is unfinished
-   * @returns true if an ingestion is unfinished, false otherwise
+   * Get the current ingestion status
+   * @returns The current ingestion status:
+   * - null: Ingestion has never been started
+   * - "unfinished": Ingestion was started but hasn't finished
+   * - "done": Ingestion has completed successfully
    */
-  public async isIngestionUnfinished(): Promise<boolean> {
-    const value = await this.get(SYSTEM_KEY_INGESTION_UNFINISHED);
-    return value !== null;
+  public async getIngestionStatus(): Promise<string | null> {
+    return await this.get(SYSTEM_KEY_INGESTION_STATUS);
   }
 
   /**
    * Mark that an ingestion has started and is unfinished
+   * Sets the ingestion status to "unfinished"
    */
   public async markIngestionStarted(): Promise<void> {
```

Comment:
```suggestion
  public async markIngestionUnfinished(): Promise<void> {
```

Seems good to maximally align our use of language.